src/device-uptime/config.py

```diff
@@ -14,19 +14,17 @@ class Config:
     CSRF_ENABLED = True
     SECRET_KEY = os.getenv("SECRET_KEY")
     DB_NAME = os.getenv("DB_NAME_PROD")
-    REGISTRY_MONGO_URI = os.getenv('REGISTRY_MONGO_GCE_URI')
-    MONITORING_MONGO_URI = os.getenv('MONITORING_MONGO_GCE_URI')
-    MONITOR_FREQUENCY_MINUTES = int(os.getenv('MONITOR_FREQUENCY_MINUTES', 60))
-    BASE_API_URL = "https://staging-platform.airqo.net/api/v1"
-    DAILY_EVENTS_URL = f"{BASE_API_URL}/devices/events"
-    DEVICE_RECENT_EVENTS_URL = f"{BASE_API_URL}/data/feeds/transform/recent"
+    REGISTRY_MONGO_URI = os.getenv("REGISTRY_MONGO_GCE_URI")
+    MONITORING_MONGO_URI = os.getenv("MONITORING_MONGO_GCE_URI")
+    MONITOR_FREQUENCY_MINUTES = int(os.getenv("MONITOR_FREQUENCY_MINUTES", 60))
+    BASE_API_URL = os.getenv("BASE_API_URL")
+    DEVICE_RECENT_EVENTS_URL = f"{BASE_API_URL}/feeds/transform/recent"
 
 
 class ProductionConfig(Config):
     DEVELOPMENT = False
-    BASE_API_URL = "https://platform.airqo.net/api/v1"
-    DAILY_EVENTS_URL = f"{BASE_API_URL}/devices/events"
-    DEVICE_RECENT_EVENTS_URL = f"{BASE_API_URL}/data/feeds/transform/recent"
+    BASE_API_URL = os.getenv("BASE_API_URL")
```

Comment:
[device uptime BASE_API_URL variable](https://github.com/airqo-platform/AirQo-api/pull/1120/commits/02e49cf12ba7701e999131223d5300a7d65c70fa)

Reply:
It's https://platform.airqo.net/api/v1/data @NoahNsimbe